commands/commands.js

```diff
@@ -35,37 +32,62 @@ export const askName = () => {
     return name;
 }
 
+const userDetailsArray = [];
 const storeData = () => {
-    const userDetails = JSON.stringify(finalData);
-    const readerFile = fs.existsSync(`${__dirname}/../files/${finalData.name}.json`, "utf-8");
+    userDetailsArray.length = 0
+    const checker = fs.existsSync(`${__dirname}/../files/${finalData.name}.json`)
+    if (checker) {
+        const content = fs.readFileSync(`${__dirname}/../files/${finalData.name}.json`, "utf-8")
+        const parsedContent = JSON.parse(content);
+        parsedContent.forEach(element => {
+            userDetailsArray.push(element);
+        });
 
-    if (!readerFile) {
-        fs.writeFileSync(`${__dirname}/../files/${finalData.name}.json`, userDetails, (err) => {
-            if (err) {
-                console.log("error")
-            }
-            console.log("\n" + "Your jokes are saved on " + chalk.blue(finalData.name) + chalk.blue(".json") + ". Please enter " + chalk.blue("view") + " to view the file.");
+        // userDetailsArray.push(content);
+        userDetailsArray.push(finalData);
```

Comment:
Why is this required???

Reply:
Answered in one of your previous comments